### PR TITLE
don't emit ctors on classes with only static members

### PR DESCRIPTION
**Pull request by martine**
Avoid emitting stub declarations on classes that only have static
members, as that helps us avoid getting calls to super() wrong.

More avoiding of issue #38.

### CODE REVIEW

File: test_files/super.tr.js [hunk 1]
```diff
@@ -59,3 +59,6 @@ class SuperTestDerivedInterface {
         // Sickle: end stub declarations.
     }
 }
+class SuperTestStaticProp extends SuperTestBaseOneArg {
+}
```
Comment (martine) [inline test_files/super.tr.js]: This bit shows why this change is necessary.  It's important here that we don't create a synthetic ctor, because we don't know the argument to pass to the base class.

**Comment by rkirov** [comment]
The less synthetic constructors the better. LGTM.
